sim/index.html

```diff
@@ -15,36 +15,36 @@
 			<div id="hide_on_first_playthrough">
 				<div id="section_dynamics">
 
-					On <i>average</i>,
-					each <icon i></icon>...
+					Im <i>Durchschnitt</i>,
+					infiziert jeder <icon i></icon>...
 					<br><br>
 					
 					<div id="label_transmission">
-						Infects 1 <icon s></icon> per <span id="label_p_transmission">N</span> days
+						1 <icon s></icon> pro <span id="label_p_transmission">N</span> Tagen
```

Comment:
Ich denke, hier sollte es besser "in" statt "pro" heißen: "Im Durschnitt infiziert jeder 🥵 1  😐 ~~pro~~ in 4 Tagen". Die Zahl "1" durch "einen" zu ersetzen würde es noch flüssiger machen.